src/main/java/com/opinta/dao/ShipmentDaoImpl.java

```diff
@@ -44,10 +44,10 @@ public Shipment getById(long id) {
         Session session = sessionFactory.getCurrentSession();
         return (Shipment) session.get(Shipment.class, id);
     }
-
     @Override
     public Shipment save(Shipment shipment) {
         Session session = sessionFactory.getCurrentSession();
+        session.save(shipment);
```

Comment:
That is how i fixed problem with hibernate, it didn't saved parcelItems. But adding fetchType.merge also fixes that so i will use it instead of addisional save.